admin/tabs/settings-config.php

```diff
@@ -0,0 +1,195 @@
+                    { "slug": "disciple-tools-genmapper", "url": "https://github.com/DiscipleTools/disciple-tools-genmapper/releases/latest/download/disciple-tools-genmapper.zip"},
+                    { "slug": "disciple-tools-import", "url": "https://github.com/DiscipleTools/disciple-tools-import/releases/latest/download/disciple-tools-import.zip"},
+                    { "slug": "disciple-tools-mobile-app-plugin", "url": "https://github.com/DiscipleTools/disciple-tools-mobile-app-plugin/releases/latest/download/disciple-tools-mobile-app-plugin.zip"},
+                    { "slug": "disciple-tools-network-dashboard", "url": "https://github.com/DiscipleTools/disciple-tools-network-dashboard/releases/latest/download/disciple-tools-network-dashboard.zip"},
+                    { "slug": "dt-outline-vpn", "url": "https://github.com/cairocoder01/dt-outline-vpn/releases/latest/download/disciple-tools-outline-vpn.zip"},
+                    { "slug": "disciple-tools-setup-wizard", "url": "https://github.com/cairocoder01/dt-setup-wizard/releases/latest/download/disciple-tools-setup-wizard.zip"},
+                    { "slug": "disciple-tools-training", "url": "https://github.com/discipletools/disciple-tools-training/releases/latest/download/disciple-tools-training.zip"}
+                ]
+            }
+        }, {
+            "name": "Wordfence",
+            "description": "Go to [Wordfence Central](https://www.wordfence.com/central) to add this site."
+        }, {
+            "name": "User Setup",
+            "description": "Select the users from the list below that should be included in this site. In addition, add the site admin user that is specific to this site.",
+            "config": {
+                "users": [{
+                    "username": "lastgen2414",
+                    "email": "lastgen2414@test.com",
+                    "roles": ["dt_admin"],
+                    "displayName": "Lauren"
+                }, {
+                    "username": "nomadicpadawan",
+                    "email": "nomadicpadawan@test.com",
+                    "roles": ["dt_admin"],
+                    "displayName": "Amber"
+                }, {
+                    "username": "write",
+                    "email": "writeussoon@test.com",
+                    "roles": ["dt_admin"],
+                    "displayName": "Phil"
+                }, {
+                    "username": "fishingwithnets",
+                    "email": "fishingwithnets@test.com",
+                    "roles": ["dt_admin"],
+                    "displayName": "James"
+                }, {
+                    "username": "mtm.data.team",
+                    "email": "mtm.data.team@test.com",
+                    "roles": ["custom_plugin_admin", "strategist"],
+                    "displayName": "MTM Data Team"
+                }]
```

Comment:
Remove all of these email addresses and replace them with fake/placeholder email addresses. We don't want them exposed since this is open source code.